src/components/two-factor-authentication/TwoFactorAuthenticationSetUp.vue

```diff
@@ -100,7 +159,7 @@
                 :block="mdAndDown"
                 class="secondary button"
                 @click.stop="close2FADialog(isActive)"
-                data-testid="close-two-way-authentification-dialog-button"
+                data-testid="close-two-factor-authentication-dialog"
```

Comment:
```suggestion
                data-testid="close-two-factor-authentication-dialog-button"
```
button ist okay, je eindeutiger, desto besser